src/semantize/typing.nit

```diff
@@ -297,6 +297,12 @@ private class TypeVisitor
 		return mclass.mclass_type
 	end
 
+	# Construction of a specific callsite according to the current context.
+	# Three entry points exist to create a callsite based on knowledge.
+	# The get_method is a top entry point, the method find the mpropdefs to call by the name of this.
+	# see `build_callsite_by_property` and `build_callsite_by_propdef` for more detail.
+	# If you know allready the mpropdef to call use directly the get_method_by_propdef method
+	# If you just know the mproperty use the build_callsite_by_property method to display error if no mpropdef is not found in the context
 	fun get_method(node: ANode, recvtype: MType, name: String, recv_is_self: Bool): nullable CallSite
```

Comment:
Still not fan of the name, it returns a `Callsite`

Reply:
I'm the culprit here, originally it returned a mmethoddef, now it returns a callsite with a lot of information.

Reply:
Now the method has been renamed to be more meaningful (build_callsite_by_name).